bin/vcf2gvf.py

```diff
@@ -84,32 +84,28 @@ def vcftogvf(var_data, strain):
             key = 'Variant_seq'
         new_df['#attributes'] = new_df['#attributes'].astype(str) + key + '=' + df[column].astype(str) + ';'
 
-    #add ao, dp, ro
+    #make 'INFO' column easier to extract attributes from
     info = df['INFO'].str.split(pat=';').apply(pd.Series) #split at ;, form dataframe
-    new_df['#attributes'] = new_df['#attributes'] + info[5].str.lower() + ';' #ao
-    new_df['#attributes'] = new_df['#attributes'] + info[7].str.lower() + ';' #dp
-    new_df['#attributes'] = new_df['#attributes'] + info[28].str.lower() + ';' #ro
+    for column in info.columns:
+        split = info[column].str.split(pat='=').apply(pd.Series)
+        title = split[0].drop_duplicates().tolist()[0].lower()
+        content = split[1]
+        info[column] = content #ignore "tag=" in column content
+        info.rename(columns={column:title}, inplace=True) #make attribute tag as column label
     
+    #add 'INFO' attributes by name
+    for column in ['ao','dp','ro']:
+        new_df['#attributes'] = new_df['#attributes'].astype(str) + column + '=' + info[column].astype(str) + ';'
+
     #add strain name
     new_df['#attributes'] = new_df['#attributes'] + 'viral_lineage=' + strain + ';'
-
-    #add WHO strain name
-    alt_strain_names = {'B.1.1.7': 'Alpha', 'B.1.351': 'Beta', 'P.1': 'Gamma', 'B.1.617.2': 'Delta', 'B.1.427': 'Epsilon', 'B.1.429': 'Epsilon', 'P.2': 'Zeta', 'B.1.525': 'Eta', 'P.3': 'Theta', 'B.1.526': 'Iota', 'B.1.617.1': 'Kappa'}
```

Comment:
@miseminger I think this dictionary needs more entries. More than 1 lineages belong to same strain name - for example Delta variant include all these - `B.1.617.2, AY.1, AY.2, AY.3`